components/org.wso2.carbon.identity.oauth2.grant.organizationswitch/src/main/java/org/wso2/carbon/identity/oauth2/grant/organizationswitch/OrganizationSwitchGrant.java

```diff
@@ -136,6 +148,24 @@ public boolean validateGrant(OAuthTokenReqMessageContext tokReqMsgCtx) throws Id
         return true;
     }
 
+    @Override
+    public OAuth2AccessTokenRespDTO issue(OAuthTokenReqMessageContext tokReqMsgCtx) throws IdentityOAuth2Exception {
+
+        // In the Asgardeo console's token system, users are given two tokens upon login - an Authorization code token
```

Comment:
Shall we update the comment without specifying Asgardeo. A generalised comment.

Reply:
Resolved via https://github.com/wso2-extensions/identity-oauth2-grant-organization-switch/pull/15